edd-discount-widget.php

```diff
@@ -55,8 +55,15 @@ function widget( $args, $instance ) {
             if( $title ) echo $before_title . $title . $after_title;
 
             $discounts = $this->get_discounts( $site_url, $api_key, $api_token, $max_discounts );
-            $discounts = json_decode( $discounts, true );
-            $discounts = $discounts['discounts'];
+
+            if ( is_wp_error( $discounts ) ) {
```

Comment:
Could we move this logic up to the beginning of the widget output, so that for users/visitors without `manage_options` would just get no widget output at all? With this where it is, the widget title is outputting, even though there are no discounts to show.

Reply:
That's a good suggestion Robin.